helpers/movimientosArkaHelper/movimientosArkaHelper.go

```diff
@@ -175,3 +230,27 @@ func PutMovimiento(movimiento *models.Movimiento, movimientoId int) (movimientoR
 	return movimientoRes, nil
 
 }
+
+func PutSoporteMovimiento(soporte *models.SoporteMovimiento, soporteId int) (soporteR *models.SoporteMovimiento, outputError map[string]interface{}) {
```

Comment:
Falta documentar, cambiar `...` en el _suggested change_, digamos "PutSoporteMovimiento hace tal cosa"
```suggestion
// PutSoporteMovimiento ...
func PutSoporteMovimiento(soporte *models.SoporteMovimiento, soporteId int) (soporteR *models.SoporteMovimiento, outputError map[string]interface{}) {
```